solid/contacts-pane
# feat(dev-folder): configured to run standalone

**Overview**
Modified so that the pane will also work on it's own.

**QA**
npm install
npm run start
change dev/index.ts addressBookToShow uri to your own.
Also, we need to make sure it doesn't break the solidos build @bourgeoa I'm not sure how to do this.
